scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -13,6 +13,7 @@
     IMAGE_NAME:                    docker.io/$IMAGE_PATH
   before_script:
     - !reference [.job-switcher, before_script]
```

Comment:
```suggestion
```
This on can be removed because it's already in `[.kubernetes-env, before_script]`